asf_public_discourse_home_decarbonisation/analysis/mse/eda_mse_category_data.py

```diff
@@ -0,0 +1,311 @@
+"""
+This script performs exploratory data analysis on Money Saving Expert data, by looking at:
+- A few statistics such as: number of posts collected, number of replies to posts and range of dates;
+- Number of posts per year;
+- Distribution of posts per user;
+- Distribution of replies per user;
+- Distribution of interactions per user.
+
+Computed statistics are logged and appear in your terminal.
```

Comment:
Might be nice to save this in a log file, with the date-time so we can potentially track changes over time in the statistics.